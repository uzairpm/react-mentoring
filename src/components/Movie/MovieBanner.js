import React, {Component} from 'react';

import PropTypeShapes from '../Common/PropTypeShapes';

const MovieBanner = (props) => {
    const { 
        poster_path,
        title,
        vote_average,
        genres,
        release_date, 
        runtime,
        overview
    } = props.movie;
    return (
        <div className="container mt-2">
            <div className="card text-white bg-dark">
                <div className="card-body">
                    <div className="row">
                        <div className="col-sm-3 col-md-3 col-lg-3">
                            <img className="albumArt" src={poster_path} />
                        </div>
                        <div className="col-sm-9 col-md-9 col-lg-9 padding25">
                            <h4 className="movieTitle mr-3">{title}</h4>
                            <span className="avgRating">{vote_average}</span>
                            <p className="text-muted">{genres && genres.join(', ')}</p>
                            <div className="mt-2">
                                <b className="mr-3">{release_date && release_date.substr(0,4)}</b>
                                <b>{runtime} mins</b>
                            </div>
                            <p className="movieDesc mt-3">{overview}</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
};
MovieBanner.propTypes = {
    movie: PropTypeShapes.moviePropType
};
MovieBanner.defaultProps = {
    vote_average: 0
};
export default MovieBanner;
