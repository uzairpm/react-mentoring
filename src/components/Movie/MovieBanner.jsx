import React from 'react';
import styled from 'styled-components';

import PropTypeShapes from '../Common/PropTypeShapes';

const PaddedDiv = styled.div`
    padding: 25px;
`;
const MovieTitle = styled.h4`
    color: #fa6c85;
    display: inline-block;
`;
const MovieDesc = styled.p`
    font-size: 1rem;
`;
const Rating = styled.span`
    border: 1px solid white;
    border-radius: 30px;
    padding: 3px 9px;
`;

const MovieBanner = (props) => {
    const { 
        poster_path,
        title,
        vote_average,
        genres,
        release_date, 
        runtime,
        overview
    } = props.movie;
    return (
        <div className="container mt-2">
            <div className="card text-white bg-dark">
                <div className="card-body">
                    <div className="row">
                        <div className="col-sm-3 col-md-3 col-lg-3">
                            <img className="img-fluid rounded" src={poster_path} />
                        </div>
                        <PaddedDiv className="col-sm-9 col-md-9 col-lg-9">
                            <MovieTitle className="mr-3">{title}</MovieTitle>
                            <Rating>{vote_average}</Rating>
                            <p className="text-muted">{genres && genres.join(', ')}</p>
                            <div className="mt-2">
                                <b className="mr-3">{release_date && release_date.substr(0,4)}</b>
                                <b>{runtime} mins</b>
                            </div>
                            <MovieDesc className="mt-3">{overview}</MovieDesc>
                        </PaddedDiv>
                    </div>
                </div>
            </div>
        </div>
    );
};
MovieBanner.propTypes = {
    movie: PropTypeShapes.moviePropType
};
MovieBanner.defaultProps = {
    vote_average: 0
};
export default MovieBanner;
