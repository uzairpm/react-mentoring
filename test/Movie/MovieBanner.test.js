import { shallow } from 'enzyme';
import React from 'react';

import MovieBanner from '../../src/components/Movie/MovieBanner';

describe('Check Component creation', () => {
    let element;
    beforeAll(() => {
        const movieObj = {
            poster_path: '',
            title: 'Feel Good',
            vote_average: 8,
            genres: ['Action', 'Drama'],
            release_date: '2017-10-10',
            runtime: 108,
            overview: 'A feel good movie'
        };
        element = shallow(<MovieBanner movie={movieObj}/>);
    });
    test('MovieBanner component snapshot test', () => {
        expect(element).toMatchSnapshot();
    });
});