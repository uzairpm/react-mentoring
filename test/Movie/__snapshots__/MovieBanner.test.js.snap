// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieBanner Component Snapshot test when a movie object is passed 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MovieBanner
    movie={
      Object {
        "genres": Array [
          "Action",
          "Drama",
        ],
        "overview": "A feel good movie",
        "poster_path": "",
        "release_date": "2017-10-10",
        "runtime": 108,
        "title": "Feel Good",
        "vote_average": 8,
      }
    }
    vote_average={0}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="card text-white bg-dark"
      >
        <div
          className="card-body"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-md-3 col-lg-3"
            >
              <img
                className="img-fluid rounded"
                src=""
              />
            </div>
            <div
              className="col-sm-9 col-md-9 col-lg-9 padding25"
              style={
                Object {
                  "padding": "25px",
                }
              }
            >
              <h4
                className="movieTitle mr-3"
                style={
                  Object {
                    "color": "#fa6c85",
                    "display": "inline-block",
                  }
                }
              >
                Feel Good
              </h4>
              <span
                className="avgRating"
                style={
                  Object {
                    "border": "1px solid white",
                    "borderRadius": "30px",
                    "padding": "3px 9px",
                  }
                }
              >
                8
              </span>
              <p
                className="text-muted"
              >
                Action, Drama
              </p>
              <div
                className="mt-2"
              >
                <b
                  className="mr-3"
                >
                  2017
                </b>
                <b>
                  108
                   mins
                </b>
              </div>
              <p
                className="movieDesc mt-3"
                style={
                  Object {
                    "fontSize": "1rem",
                  }
                }
              >
                A feel good movie
              </p>
            </div>
          </div>
        </div>
      </div>,
      "className": "container mt-2",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="card-body"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-md-3 col-lg-3"
            >
              <img
                className="img-fluid rounded"
                src=""
              />
            </div>
            <div
              className="col-sm-9 col-md-9 col-lg-9 padding25"
              style={
                Object {
                  "padding": "25px",
                }
              }
            >
              <h4
                className="movieTitle mr-3"
                style={
                  Object {
                    "color": "#fa6c85",
                    "display": "inline-block",
                  }
                }
              >
                Feel Good
              </h4>
              <span
                className="avgRating"
                style={
                  Object {
                    "border": "1px solid white",
                    "borderRadius": "30px",
                    "padding": "3px 9px",
                  }
                }
              >
                8
              </span>
              <p
                className="text-muted"
              >
                Action, Drama
              </p>
              <div
                className="mt-2"
              >
                <b
                  className="mr-3"
                >
                  2017
                </b>
                <b>
                  108
                   mins
                </b>
              </div>
              <p
                className="movieDesc mt-3"
                style={
                  Object {
                    "fontSize": "1rem",
                  }
                }
              >
                A feel good movie
              </p>
            </div>
          </div>
        </div>,
        "className": "card text-white bg-dark",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
          >
            <div
              className="col-sm-3 col-md-3 col-lg-3"
            >
              <img
                className="img-fluid rounded"
                src=""
              />
            </div>
            <div
              className="col-sm-9 col-md-9 col-lg-9 padding25"
              style={
                Object {
                  "padding": "25px",
                }
              }
            >
              <h4
                className="movieTitle mr-3"
                style={
                  Object {
                    "color": "#fa6c85",
                    "display": "inline-block",
                  }
                }
              >
                Feel Good
              </h4>
              <span
                className="avgRating"
                style={
                  Object {
                    "border": "1px solid white",
                    "borderRadius": "30px",
                    "padding": "3px 9px",
                  }
                }
              >
                8
              </span>
              <p
                className="text-muted"
              >
                Action, Drama
              </p>
              <div
                className="mt-2"
              >
                <b
                  className="mr-3"
                >
                  2017
                </b>
                <b>
                  108
                   mins
                </b>
              </div>
              <p
                className="movieDesc mt-3"
                style={
                  Object {
                    "fontSize": "1rem",
                  }
                }
              >
                A feel good movie
              </p>
            </div>
          </div>,
          "className": "card-body",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col-sm-3 col-md-3 col-lg-3"
              >
                <img
                  className="img-fluid rounded"
                  src=""
                />
              </div>,
              <div
                className="col-sm-9 col-md-9 col-lg-9 padding25"
                style={
                  Object {
                    "padding": "25px",
                  }
                }
              >
                <h4
                  className="movieTitle mr-3"
                  style={
                    Object {
                      "color": "#fa6c85",
                      "display": "inline-block",
                    }
                  }
                >
                  Feel Good
                </h4>
                <span
                  className="avgRating"
                  style={
                    Object {
                      "border": "1px solid white",
                      "borderRadius": "30px",
                      "padding": "3px 9px",
                    }
                  }
                >
                  8
                </span>
                <p
                  className="text-muted"
                >
                  Action, Drama
                </p>
                <div
                  className="mt-2"
                >
                  <b
                    className="mr-3"
                  >
                    2017
                  </b>
                  <b>
                    108
                     mins
                  </b>
                </div>
                <p
                  className="movieDesc mt-3"
                  style={
                    Object {
                      "fontSize": "1rem",
                    }
                  }
                >
                  A feel good movie
                </p>
              </div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  className="img-fluid rounded"
                  src=""
                />,
                "className": "col-sm-3 col-md-3 col-lg-3",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "img-fluid rounded",
                  "src": "",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h4
                    className="movieTitle mr-3"
                    style={
                      Object {
                        "color": "#fa6c85",
                        "display": "inline-block",
                      }
                    }
                  >
                    Feel Good
                  </h4>,
                  <span
                    className="avgRating"
                    style={
                      Object {
                        "border": "1px solid white",
                        "borderRadius": "30px",
                        "padding": "3px 9px",
                      }
                    }
                  >
                    8
                  </span>,
                  <p
                    className="text-muted"
                  >
                    Action, Drama
                  </p>,
                  <div
                    className="mt-2"
                  >
                    <b
                      className="mr-3"
                    >
                      2017
                    </b>
                    <b>
                      108
                       mins
                    </b>
                  </div>,
                  <p
                    className="movieDesc mt-3"
                    style={
                      Object {
                        "fontSize": "1rem",
                      }
                    }
                  >
                    A feel good movie
                  </p>,
                ],
                "className": "col-sm-9 col-md-9 col-lg-9 padding25",
                "style": Object {
                  "padding": "25px",
                },
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Feel Good",
                    "className": "movieTitle mr-3",
                    "style": Object {
                      "color": "#fa6c85",
                      "display": "inline-block",
                    },
                  },
                  "ref": null,
                  "rendered": "Feel Good",
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 8,
                    "className": "avgRating",
                    "style": Object {
                      "border": "1px solid white",
                      "borderRadius": "30px",
                      "padding": "3px 9px",
                    },
                  },
                  "ref": null,
                  "rendered": 8,
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Action, Drama",
                    "className": "text-muted",
                  },
                  "ref": null,
                  "rendered": "Action, Drama",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <b
                        className="mr-3"
                      >
                        2017
                      </b>,
                      <b>
                        108
                         mins
                      </b>,
                    ],
                    "className": "mt-2",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "2017",
                        "className": "mr-3",
                      },
                      "ref": null,
                      "rendered": "2017",
                      "type": "b",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          108,
                          " mins",
                        ],
                      },
                      "ref": null,
                      "rendered": Array [
                        108,
                        " mins",
                      ],
                      "type": "b",
                    },
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "A feel good movie",
                    "className": "movieDesc mt-3",
                    "style": Object {
                      "fontSize": "1rem",
                    },
                  },
                  "ref": null,
                  "rendered": "A feel good movie",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="card text-white bg-dark"
        >
          <div
            className="card-body"
          >
            <div
              className="row"
            >
              <div
                className="col-sm-3 col-md-3 col-lg-3"
              >
                <img
                  className="img-fluid rounded"
                  src=""
                />
              </div>
              <div
                className="col-sm-9 col-md-9 col-lg-9 padding25"
                style={
                  Object {
                    "padding": "25px",
                  }
                }
              >
                <h4
                  className="movieTitle mr-3"
                  style={
                    Object {
                      "color": "#fa6c85",
                      "display": "inline-block",
                    }
                  }
                >
                  Feel Good
                </h4>
                <span
                  className="avgRating"
                  style={
                    Object {
                      "border": "1px solid white",
                      "borderRadius": "30px",
                      "padding": "3px 9px",
                    }
                  }
                >
                  8
                </span>
                <p
                  className="text-muted"
                >
                  Action, Drama
                </p>
                <div
                  className="mt-2"
                >
                  <b
                    className="mr-3"
                  >
                    2017
                  </b>
                  <b>
                    108
                     mins
                  </b>
                </div>
                <p
                  className="movieDesc mt-3"
                  style={
                    Object {
                      "fontSize": "1rem",
                    }
                  }
                >
                  A feel good movie
                </p>
              </div>
            </div>
          </div>
        </div>,
        "className": "container mt-2",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="card-body"
          >
            <div
              className="row"
            >
              <div
                className="col-sm-3 col-md-3 col-lg-3"
              >
                <img
                  className="img-fluid rounded"
                  src=""
                />
              </div>
              <div
                className="col-sm-9 col-md-9 col-lg-9 padding25"
                style={
                  Object {
                    "padding": "25px",
                  }
                }
              >
                <h4
                  className="movieTitle mr-3"
                  style={
                    Object {
                      "color": "#fa6c85",
                      "display": "inline-block",
                    }
                  }
                >
                  Feel Good
                </h4>
                <span
                  className="avgRating"
                  style={
                    Object {
                      "border": "1px solid white",
                      "borderRadius": "30px",
                      "padding": "3px 9px",
                    }
                  }
                >
                  8
                </span>
                <p
                  className="text-muted"
                >
                  Action, Drama
                </p>
                <div
                  className="mt-2"
                >
                  <b
                    className="mr-3"
                  >
                    2017
                  </b>
                  <b>
                    108
                     mins
                  </b>
                </div>
                <p
                  className="movieDesc mt-3"
                  style={
                    Object {
                      "fontSize": "1rem",
                    }
                  }
                >
                  A feel good movie
                </p>
              </div>
            </div>
          </div>,
          "className": "card text-white bg-dark",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row"
            >
              <div
                className="col-sm-3 col-md-3 col-lg-3"
              >
                <img
                  className="img-fluid rounded"
                  src=""
                />
              </div>
              <div
                className="col-sm-9 col-md-9 col-lg-9 padding25"
                style={
                  Object {
                    "padding": "25px",
                  }
                }
              >
                <h4
                  className="movieTitle mr-3"
                  style={
                    Object {
                      "color": "#fa6c85",
                      "display": "inline-block",
                    }
                  }
                >
                  Feel Good
                </h4>
                <span
                  className="avgRating"
                  style={
                    Object {
                      "border": "1px solid white",
                      "borderRadius": "30px",
                      "padding": "3px 9px",
                    }
                  }
                >
                  8
                </span>
                <p
                  className="text-muted"
                >
                  Action, Drama
                </p>
                <div
                  className="mt-2"
                >
                  <b
                    className="mr-3"
                  >
                    2017
                  </b>
                  <b>
                    108
                     mins
                  </b>
                </div>
                <p
                  className="movieDesc mt-3"
                  style={
                    Object {
                      "fontSize": "1rem",
                    }
                  }
                >
                  A feel good movie
                </p>
              </div>
            </div>,
            "className": "card-body",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-sm-3 col-md-3 col-lg-3"
                >
                  <img
                    className="img-fluid rounded"
                    src=""
                  />
                </div>,
                <div
                  className="col-sm-9 col-md-9 col-lg-9 padding25"
                  style={
                    Object {
                      "padding": "25px",
                    }
                  }
                >
                  <h4
                    className="movieTitle mr-3"
                    style={
                      Object {
                        "color": "#fa6c85",
                        "display": "inline-block",
                      }
                    }
                  >
                    Feel Good
                  </h4>
                  <span
                    className="avgRating"
                    style={
                      Object {
                        "border": "1px solid white",
                        "borderRadius": "30px",
                        "padding": "3px 9px",
                      }
                    }
                  >
                    8
                  </span>
                  <p
                    className="text-muted"
                  >
                    Action, Drama
                  </p>
                  <div
                    className="mt-2"
                  >
                    <b
                      className="mr-3"
                    >
                      2017
                    </b>
                    <b>
                      108
                       mins
                    </b>
                  </div>
                  <p
                    className="movieDesc mt-3"
                    style={
                      Object {
                        "fontSize": "1rem",
                      }
                    }
                  >
                    A feel good movie
                  </p>
                </div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    className="img-fluid rounded"
                    src=""
                  />,
                  "className": "col-sm-3 col-md-3 col-lg-3",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "img-fluid rounded",
                    "src": "",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h4
                      className="movieTitle mr-3"
                      style={
                        Object {
                          "color": "#fa6c85",
                          "display": "inline-block",
                        }
                      }
                    >
                      Feel Good
                    </h4>,
                    <span
                      className="avgRating"
                      style={
                        Object {
                          "border": "1px solid white",
                          "borderRadius": "30px",
                          "padding": "3px 9px",
                        }
                      }
                    >
                      8
                    </span>,
                    <p
                      className="text-muted"
                    >
                      Action, Drama
                    </p>,
                    <div
                      className="mt-2"
                    >
                      <b
                        className="mr-3"
                      >
                        2017
                      </b>
                      <b>
                        108
                         mins
                      </b>
                    </div>,
                    <p
                      className="movieDesc mt-3"
                      style={
                        Object {
                          "fontSize": "1rem",
                        }
                      }
                    >
                      A feel good movie
                    </p>,
                  ],
                  "className": "col-sm-9 col-md-9 col-lg-9 padding25",
                  "style": Object {
                    "padding": "25px",
                  },
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Feel Good",
                      "className": "movieTitle mr-3",
                      "style": Object {
                        "color": "#fa6c85",
                        "display": "inline-block",
                      },
                    },
                    "ref": null,
                    "rendered": "Feel Good",
                    "type": "h4",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": 8,
                      "className": "avgRating",
                      "style": Object {
                        "border": "1px solid white",
                        "borderRadius": "30px",
                        "padding": "3px 9px",
                      },
                    },
                    "ref": null,
                    "rendered": 8,
                    "type": "span",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Action, Drama",
                      "className": "text-muted",
                    },
                    "ref": null,
                    "rendered": "Action, Drama",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <b
                          className="mr-3"
                        >
                          2017
                        </b>,
                        <b>
                          108
                           mins
                        </b>,
                      ],
                      "className": "mt-2",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "2017",
                          "className": "mr-3",
                        },
                        "ref": null,
                        "rendered": "2017",
                        "type": "b",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            108,
                            " mins",
                          ],
                        },
                        "ref": null,
                        "rendered": Array [
                          108,
                          " mins",
                        ],
                        "type": "b",
                      },
                    ],
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "A feel good movie",
                      "className": "movieDesc mt-3",
                      "style": Object {
                        "fontSize": "1rem",
                      },
                    },
                    "ref": null,
                    "rendered": "A feel good movie",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
